code/game/objects/items/weapons/tanks/tanks.dm

```diff
@@ -65,7 +65,11 @@ var/list/global/tank_gauge_cache = list()
 	src.air_contents = new /datum/gas_mixture()
 	src.air_contents.volume = volume //liters
 	src.air_contents.temperature = T20C
+<<<<<<< HEAD
 	processing_objects.Add(src)
+=======
+	START_PROCESSING(SSobj, src)

```

Comment:
The hell is going on here? I didn't make this change.

Reply:
That change was part of a refactor done upstream.